src/solver/types.jl

```diff
@@ -126,10 +126,12 @@ Base.@kwdef struct ViscousSponge{FT} <: AbstractSponge
 end
 
 Base.@kwdef struct RayleighSponge{FT} <: AbstractSponge
+    zmax::FT
```

Comment:
I don't think `zmax` should be in here. `zmax` is a property of the domain, one should be able to define a `RayleighSponge` independently on the computational grid used